src/bin/cargo/commands/owner.rs

```diff
@@ -6,24 +6,74 @@ use cargo_credential::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg(Arg::new("crate").value_name("CRATE").action(ArgAction::Set))
+        .arg(Arg::new("crate").hide(true))
+        .arg_required_else_help(true)
+        .args_conflicts_with_subcommands(true)
+        .override_usage(
+            "\
+       cargo owner add    <OWNER_NAME> [CRATE_NAME] [OPTIONS]
```

Comment:
btw if we update our version of clap, you can set `flatten_help(true)` and it will give you this usage automatically

Reply:
OK, I will replace it in time after clap is updated.

Reply:
Feel free to update clap as part of your PR

Reply:
It has been modified.